ikarus/finiteelements/mechanics/kirchhoffloveshell.hh

```diff
@@ -216,65 +388,45 @@ namespace Ikarus {
      * @return The calculated scalar value.
      */
     template <typename ScalarType>
-    auto calculateScalarImpl(const FERequirementType& par, const std::optional<const Eigen::VectorX<ScalarType>>& dx
+    auto calculateScalarImpl(const FERequirementType &par, const std::optional<const Eigen::VectorX<ScalarType>> &dx
```

Comment:
doxygen comment not needed for `calculateScalarImpl` function as it is already provided for `calculateScalar`. If needed, it should be consistently added also to `calculateVectorImpl` and `calculateMatrixImpl`